products/Zephyr Squad/README.md

```diff
@@ -0,0 +1,9 @@
+This file contains metadata used to specify what APIs to link to a product and what section order documentation will follow.
+
+## Content Metadata
+
+| Order | Name | Type | ContentURL |
+| ----- | ---- | ---- | ---------- |
+| 0 | Introduction | `markdown` | ./Introduction.md |
+| 1 | Authentication | `markdown` | ./Authentication.md |
+| 2 | Zephyr Squad API | `apiURL` | https://app.swaggerhub.com/apis/smartbear/zephyr-squad_cloud_api/2 |
```

Comment:
FYI - this URL will change but I can do that once the API is published in to our `smartbear-public` org in SwaggerHub